### PR TITLE
migrate from enzyme to testing-library

### PR DESCRIPTION
**Pull request by akellbl4**
Airbnb stopped support of Enzyme and we have to migrate to Testing Library as a more reliable tool with strong support from the community.

### CODE REVIEW

**Comment by esvyridov** [comment]
@akellbl4 hi, can I try to contribute to this PR as well?

**Comment by paskal** [comment]
@esvyridov feel free to, but please create your own PR for that.

**Comment by akellbl4** [comment]
@esvyridov hey, I've merged master into this branch. the best way to help with this PR is to create branch from it and open PR to this branch. Once we have all tests we are ready to merge this PR as a whole.

**Comment by esvyridov** [comment]
@akellbl4 ok, got it, thanks 👍

File: frontend/app/components/comment-form/__subscribe-by-email/comment-form__subscribe-by-email.test.tsx [hunk 1]
```diff
@@ -19,38 +20,20 @@ import enMessages from 'locales/en.json';
...
-const emailVerificationForSubscribeMock = (emailVerificationForSubscribe as unknown) as jest.Mock<
-  ReturnType<typeof emailVerificationForSubscribe>
->;
-const emailConfirmationForSubscribeMock = (emailConfirmationForSubscribe as unknown) as jest.Mock<
-  ReturnType<typeof emailConfirmationForSubscribe>
->;
-const unsubscribeFromEmailUpdatesMock = (unsubscribeFromEmailUpdates as unknown) as jest.Mock<
-  ReturnType<typeof unsubscribeFromEmailUpdates>
->;
+import { debug } from 'webpack';
```
Comment (Mavrin) [inline frontend/app/components/comment-form/__subscribe-by-email/comment-form__subscribe-by-email.test.tsx:23]: unused import

File: frontend/app/components/comment-form/__subscribe-by-email/comment-form__subscribe-by-email.test.tsx [hunk 2]
```diff
@@ -88,105 +71,84 @@ describe('<SubscribeByEmailForm/>', () => {
...
-
-    const input = wrapper.find('input');
-    const form = wrapper.find('form');
-
-    input.getDOMNode<HTMLInputElement>().value = 'some@email.com';
-    input.simulate('input');
-    form.simulate('submit');
-
-    expect(emailVerificationForSubscribeMock).toHaveBeenCalledWith('some@email.com');
+    const { container, getByRole, getByPlaceholderText, getByText, debug } = createContainer();
```
Comment (Mavrin) [inline frontend/app/components/comment-form/__subscribe-by-email/comment-form__subscribe-by-email.test.tsx:93]: `debug` unused var

File: frontend/app/components/comment-form/__subscribe-by-email/comment-form__subscribe-by-email.test.tsx [hunk 3]
```diff
@@ -88,105 +71,84 @@ describe('<SubscribeByEmailForm/>', () => {
...
-
-    expect(wrapper.text().startsWith('You have been unsubscribed by email to updates')).toBe(true);
-    expect(wrapper.find(Button).text()).toEqual('Close');
+    expect(emailConfirmationForSubscribe).toHaveBeenCalledWith('tokentokentoken');
+    expect(container.querySelector('.comment-form__subscribe-by-email')).toHaveTextContent(
+      'You have been subscribed on updates by email'
+    );
+    expect(getByRole('button')).toEqual('Unsubscribe');
   });
+  //   it('should fill in email from local storage', async () => {
```
Comment (Mavrin) [inline frontend/app/components/comment-form/__subscribe-by-email/comment-form__subscribe-by-email.test.tsx:119]: Need restore this test or remove at all

**Pull request closed without merging**